### PR TITLE
Feature: Xcode 15.3 support

### PR DESCRIPTION
**Pull request by Brett-Best**
Closes https://github.com/CocoaPods/Xcodeproj/issues/933

### CODE REVIEW

**Comment by Whirlwind** [comment]
Any process?

**Comment by FilDevTronic** [comment]
@amorde @Whirlwind can someone please have a look at this MR that's ready?

**Comment by cci-mzdogan** [comment]
Waiting for the update pls. @amorde  @Whirlwind

File: lib/xcodeproj/project/object/root_object.rb [hunk 1]
```diff
@@ -49,6 +49,10 @@ class PBXProject < AbstractObject
         #
         has_one :main_group, PBXGroup
 
+        #  @return [String] whether minimizes project reference proxies
```
[the remark this answers is not in the document]
Comment (Brett-Best) [reply lib/xcodeproj/project/object/root_object.rb:52]: Done thanks!

**Comment by Kaspik** [comment]
Are you planning to release an updated version of Xcodeproj with this fix?

**Pull request merged**